cime/config/e3sm/config_grids.xml

```diff
@@ -1237,6 +1187,16 @@
       <mask>gx1v6</mask>
     </model_grid>
 
+    <model_grid alias="ne120pg2_r05_oECv3">
```

Comment:
Normally the ne120 atmosphere is run with the oRRS18v3 ocean grid.  Is there a reason you are using oECv3?

Reply:
I don't know what people use. I mainly just wanted to do some ne120 hindcasts with fixed SST and not have to spin up a new r05 land initial condition. 
Should I just remove this grid?

Reply:
Yes go ahead.

Reply:
There's a ne120pg2_ne120pg2 grid you could use for fixed-SST runs.  Not sure about land initial conditions for that.

Reply:
I wanted a tri-grid case to not have to change the land.

Reply:
If this grid works for you, go ahead and keep it.